### PR TITLE
[Ch333] UI for bootstrapping on first app load

### PR DESCRIPTION
**Pull request by jnu**
Currently the app can't be used without running our `python database.py --dummy-data` init script. This PR fixes that by adding a simple bootstrapping UI.

It works like this:
1) When the client makes a request to the API, the API determines if it's in the "blank slate" condition by checking if there's anything in the database (currently just looks at the `organization` table -- if that's empty, nothing has been setup). [Also, if the database does not exist at this point it will now be created automatically, with the correct tables. Things like default roles and categories will also be created here.]
2) If the app is a blank slate, the API will throw a `418` error. The client handles this by displaying the bootstrapping form.
3) The user can enter the organization name, their name, and their email address and press submit.
4) The API creates the Organization and an admin User with the given info, and emails a temporary password to the user
5) User can now log in. The app is no longer in the blank slate mode so permissions are now buttoned up as usual.

image

### CODE REVIEW

**Comment by jnu** [comment]
I also cleaned up some of the current database connections stuff, in preparation for further reworking in https://app.clubhouse.io/stanford-computational-policy-lab/story/302/use-async-db-calls

File: api/user.py [hunk 1]
```diff
@@ -196,4 +208,38 @@ async def authenticate(self, credentials) -> Optional[UserDBModel]:
...
-user_db = SQLAlchemyORMUserDatabase(database.SessionLocal)
+
+user_db = SQLAlchemyORMUserDatabase(database.connection)
+
+
+# remove cookie_secure=False later for production
+cookie_authentication = CookieAuthentication(
+        secret=settings.secret,
+        lifetime_seconds=3600,
+        cookie_secure=False,
```
Comment (seerocode) [inline api/user.py:219]: Not part of this PR but is it okay to leave the cookie security as false? I know we have a story for this but thinking for handoff.

**Review by seerocode — APPROVED** [review]
I was wondering how we were going to set up a first time user. Looks good to me! 

🎉 tests!

File: api/user.py (hunk 1 above)
Comment (jnu) [reply api/user.py:219]: it's not a big deal, i can add a config option tho
